packages/zipkin/index.d.ts

```diff
@@ -30,7 +30,7 @@ declare namespace zipkin {
   }
 
   class Tracer {
-    constructor(args: { ctxImpl: Context<TraceId>, recorder: Recorder, sampler?: sampler.Sampler, traceId128Bit?: boolean });
+    constructor(args: { ctxImpl: Context<TraceId>, recorder: Recorder, sampler?: sampler.Sampler, traceId128Bit?: boolean, localServiceName?: string, localEndpoint?: model.Endpoint });
```

Comment:
@yonran is it correct typscript to do `model.Endpoint` as I am here (to reference a type under a different namespace in the same file)?

Reply:
Yes, tsc is happy (`node_modules/.bin/tsc --strict index.d.ts`), and Visual Studio Code can jump to the correct declaration.